### ISSUE TITLE
Update Starlette

### ISSUE THREAD

**Issue by elsimqoz** [mfreeborn/fastapi-sqlalchemy#4]
@mfreeborn , it will be great to update Starlette to 0.13.2
https://github.com/encode/starlette/blob/master/starlette/__init__.py

**Comment by mfreeborn** [comment]
I'd be happy to accept a pull request if you wanted to do that :)

**Comment by elsimqoz** [comment]
https://github.com/mfreeborn/fastapi-sqlalchemy/pull/5

**Comment by mfreeborn** [comment]
Thanks you, resolved in #8

**Comment by elsimqoz** [comment]
https://pypi.org/project/FastAPI-SQLAlchemy/#history
```
FastAPI-SQLAlchemy 0.1.3 (Released: Mar 21, 2020)
```

https://github.com/mfreeborn/fastapi-sqlalchemy/commit/8672d221f33ee43ef96b5a5a7199e48af8f547f4#diff-42182d5fec0302bd1f5cf60a2f4b5b5c
```
__version__ = "0.1.4"
```
@mfreeborn it seems that there is old version on pypi
